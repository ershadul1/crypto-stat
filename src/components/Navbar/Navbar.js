import { Link } from 'react-router-dom';
import styles from './Navbar.module.css';

const Navbar = () => (
  <nav className={styles.navbar}>
    <div className={styles.container}>
      <Link className={styles.navlink} to="/">Crypto Stat</Link>
    </div>
  </nav>
);

export default Navbar;
