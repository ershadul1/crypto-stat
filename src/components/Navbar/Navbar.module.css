@import url('https://fonts.googleapis.com/css2?family=Lemonada:wght@700&display=swap');

.navbar {
  background-color: rgb(28, 63, 130);
}

.container {
  max-width: 1200px;
  margin: auto;
  padding: 30px;
}

.navlink {
  margin: auto;
  display: block;
  color: #fff;
  text-align: center;
  font-family: 'Lemonada', cursive;
  text-decoration: none;
  font-size: 30px;
}
