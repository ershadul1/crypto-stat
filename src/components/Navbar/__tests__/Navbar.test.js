import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../Navbar';

it('Navbar snapshot test', () => {
  const tree = renderer
    .create(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});