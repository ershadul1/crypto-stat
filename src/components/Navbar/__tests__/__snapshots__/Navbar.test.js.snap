// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot test 1`] = `
<nav
  className="navbar"
>
  <div
    className="container"
  >
    <a
      className="navlink"
      href="/"
      onClick={[Function]}
    >
      Crypto Stat
    </a>
  </div>
</nav>
`;
